models/organization.py

```diff
@@ -115,7 +116,7 @@ def sort_organizations(organizations):
         if organization is not None:
             number_ref.append(organization.reference)
     if number_ref:
-        number_ref = sorted(number_ref, key=int)
+        number_ref = sorted(number_ref, key=lambda ref: int(ref) if ref.isdigit() else ref)
```

Comment:
I didn't get this change either. Is it related to the problem ?

Reply:
In the data I use in local I didn't know if ref was a number or a string so I created a new organization with a string as reference. This error was preventing me to access the information page too.

We should probably enforce admin to use int when encoding reference for an organization and drop this change.